Description: pkill -u error parsing
 pkill -u wouldn't work with a space between the u and uid. This was due
to an off-by-one problem.

Author: Alfredo Esteban <aedelatorre@gmail.com>
Bug-Debian: http://bugs.debian.org/676239
--- a/pgrep.c
+++ b/pgrep.c
@@ -564,7 +564,7 @@
 {
 	int sig;
 	int i = 1;
-	while (i < *argc) {
+	if (*argc > 2 && argv[1][0] == '-') {
 		sig = signal_name_to_number(argv[i] + 1);
 		if (sig == -1 && isdigit(argv[1][1]))
 			sig = atoi(argv[1] + 1);
